k8s/helm-chart/dynatrace-gcp-monitor/values.yaml

```diff
@@ -124,9 +124,162 @@ metricResources:
+    # Google Cloud Functions
+    # - cloud_function
+    # Google Cloud IoT Core
+    # - cloudiot_device_registry
+    # Google Cloud Router
+    # - nat_gateway
+    # Google Cloud Run.
+    # - cloud_run_revision
+    # - cloud_run_job
+    # Google Cloud Storage
+    # - gcs_bucket
+    # Google Cloud Tasks
+    # - cloud_tasks_queue
+    # Google Cloud Composer
+    # - cloud_composer_environment
+    # Google Compute Engine
+    # - instance_group
+    # - gce_instance
+    # - aws_ec2_instance
+    # - tpu_worker
+    # - autoscaler
+    # - baremetalsolution.googleapis.com/Instance
+    # Google Cloud Storage Transfer
+    # - transfer_service_agent
+    # Google Dataflow
+    # - dataflow_job
+    # Google Dataproc
+    # - cloud_dataproc_cluster
+    # Google Firestore in Datastore mode
+    # - datastore_request
+    # Google Cloud Data Loss Prevention
+    # - cloud_dlp_project
+    # Google Cloud Filestore
+    # - filestore_instance
+    # Google Firebase
+    # - firebase_domain
+    # - firebase_namespace
+    # Google Cloud Firestore
+    # - firestore.googleapis.com/Database
+    # - firestore_instance
+    # Google Hybrid Connectivity
+    # - interconnect
+    # - vpn_gateway
+    # - interconnect_attachment
+    # - gce_router
+    # Google Kubernetes Engine (GKE)
+    #  - istio_control_plane
+    # - k8s_node
+    # - k8s_pod
+    # - k8s_container
+    # Google Cloud Load Balancing
+    # - loadbalancing.googleapis.com/ExternalNetworkLoadBalancerRule
+    # - udp_lb_rule
+    # - internal_tcp_lb_rule
+    # - internal_http_lb_rule
+    # - internal_udp_lb_rule
+    # - tcp_lb_rule
+    # - https_lb_rule
+    # - tcp_ssl_proxy_rule
+    # Google Managed Microsoft AD
+    # - microsoft_ad_domain
+    # Google Memorystore
+    # - redis_instance
+    # NetApp on Google Cloud
+    # - netapp_cloud_volume
+    # - cloudvolumesgcp-api.netapp.com/CloudVolume
+    # Google Network Security
+    # - network_security_policy
+    # Google Cloud's operations suite (disabled)
+    #  servicedirectory_service
+    #  k8s_service
+    #  logging_sink
+    #  aws_elb_load_balancer
+    #  aws_ec2_instance
+    #  gae_app
+    #  cloud_run_revision
+    #  cloudtrace.googleapis.com/CloudtraceProject
+    #  gce_instance
+    #  uptime_url
+    # Google Pub/Sub
+    # - pubsub_snapshot
+    # - pubsub_subscription
+    # - pubsub_topic
+    # Google Pub/Sub Lite
+    # - pubsublite_subscription_partition
+    # - pubsublite_topic_partition
+    # Google reCAPTCHA Enterprise
+    # - recaptchaenterprise.googleapis.com/Key
+    # Google Cloud Spanner
+    # - spanner_instance
+    # Google Cloud SQL
+    # - cloudsql_database
+    # - cloudsql_instance_database
+    # Google Virtual Private Cloud
+    # - vpc_access_connector
+    # Google Network Topology
+    # - gce_zone_network_health
+
+# configuration file for metrics that contains names bellow that shoud not be included
+# in autodiscovery process
+autodiscoveryBlockListYaml: |
```

Comment:
Same thing with block_list. Also, we could maybe avoid passing them as parameters through all the methods, because it's a long flow. If it's only some filtering-out list, it will be easier to read all methods if we remove those params and just get the list when needed using config.py.